app/src/main/java/com/tari/android/wallet/ui/activity/home/HomeActivity.kt

```diff
@@ -648,6 +649,7 @@ class HomeActivity : BaseActivity(),
      */
     override fun onTxSelected(tx: Tx) {
         Logger.i("Transaction with id ${tx.id} selected.")
+        startActivity(TransactionDetailActivity.createIntent(this, tx))
```

Comment:
@CjS77 I'm not sure, home is more like a dashboard, so it doesn't look wrong to me to have a separate activity. But then there's [this](https://stackoverflow.com/a/54025780/597268), and [this](https://developer.android.com/guide/navigation/) and [this](https://developer.android.com/guide/navigation/navigation-getting-started). I'll have a clear answer today after going through some of these.

Reply:
I think it's good to have it defined as an activity, it will allow us to use it anywhere without much efforts. For example, in future, if we have notifications when someone receives tari, we can link this activity to it, so it can be displayed on notification tap.

Reply:
We're clearly not single activity right now :)

@jigneshsanghani makes a valid point about the other places this could be used, so I'm happy with that justification for now

Reply:
@CjS77 @jigneshsanghani ok will have a quick look at the rest of it